.travis.yml

```diff
@@ -11,11 +11,32 @@ matrix:
     - go: master
 
 install:
+  - sudo apt-get update -qq && sudo apt-get install -y -qq apache2-utils php5
```

Comment:
`php5` 😱

Reply:
it is needed for the test which checks grafana user creation.
test - `check updated grafana user`

Reply:
Where did you get https://github.com/percona/pmm-manage/blob/master/tests/password-helper.php from? Or did you wrote it?

Reply:
the file has the original header with links to the sources

Reply:
I was just looking for LICENSE confirmation https://github.com/defuse/password-hashing/blob/bcbcd8a1e580e1b73edd9e1d35902df2ebe65fdb/PasswordHash.php#L2-L28 https://github.com/defuse/password-hashing/blob/bcbcd8a1e580e1b73edd9e1d35902df2ebe65fdb/PasswordHash.php#L87-L139
Looks good then. Thanks!

Reply:
I have added license inside php script